include/solvers/gls_sharp_navier_stokes.h

```diff
@@ -248,7 +257,10 @@ class GLSSharpNavierStokesSolver : public GLSNavierStokesSolver<dim>
 private:
   std::map<unsigned int,
            std::set<typename DoFHandler<dim>::active_cell_iterator>>
-                               vertices_to_cell;
+    vertices_to_cell;
+  std::map<typename DoFHandler<dim>::active_cell_iterator,
+           std::tuple<bool, unsigned int>>
```

Comment:
Why is the element a tuple instead of just a bool? What is this unsigned int parameter for? Can you document it above the map?